external/docker-integration-tests/src/test/resources/mariadb_docker_entrypoint.sh

```diff
@@ -18,7 +18,7 @@
 
 dpkg-divert --add /bin/systemctl && ln -sT /bin/true /bin/systemctl
 apt update
-apt install -y mariadb-plugin-gssapi-server=1:10.5.5+maria~focal
+apt install -y mariadb-plugin-gssapi-server=$(dpkg -s mariadb-server | sed -n "s/^Version: \(.*\)/\1/p")
```

Comment:
nit: How about logging which version is  installed like this?
```
XXXXX=$(dpkg -s mariadb-server | sed -n "s/^Version: \(.*\)/\1/p")
echo ".... $XXXXX...."
apt install -y mariadb-plugin-gssapi-server=${XXXXXX}
```

Reply:
Seems good idea.

Reply:
The version can be seen under any circumstances in the log but doesn't hurt to add.
When works fine:
```
Preparing to unpack .../6-mariadb-plugin-gssapi-server_1%3a10.5.8+maria~focal_amd64.deb ...
```
When failed:
```
E: Version '1:10.5.5+maria~focal' for 'mariadb-plugin-gssapi-server' was not found
```

Reply:
Yeah, apt itself logs the version but I wonder if it's better to log what it is trying to install explicitly for clarity.